dune_api_scripts/update_appdata_view.py

```diff
@@ -25,13 +25,14 @@ class Environment(Enum):
 
     STAGING = "barn"
     PRODUCTION = "prod"
+    TEST = "test"
 
     def __str__(self) -> str:
         return self.value
 
     def as_query_param(self) -> QueryParameter:
         """Converts Environment to Dune Query Parameter"""
-        return QueryParameter.enum_type("Environment", self.value, ["barn", "prod"])
+        return QueryParameter.enum_type("Environment", self.value, ["barn", "prod", "test"])
```

Comment:
I wonder if there is some way for the Enum class to generate this list. Best I found was this:

```
[e.value for e in Environment]
```

What do you think about including that here. Then future devs won't have to update two independent places.